.ci/build-win.sh

```diff
@@ -20,7 +20,7 @@ meson setup \
 	--cross-file=.ci/win${ARCH}-cross-file.txt \
 	--wrap-mode=default \
 	--strip \
-	-Dtests=disabled \
+	-Dtests=enabled \
```

Comment:
Does it matter? We can’t run the tests anyway. We already run tests in MSVC and msys builds which build natively on Windows.